libobs/util/platform-nix.c

```diff
@@ -296,23 +295,12 @@ int os_get_config_path(char *dst, size_t size, const char *name)
 		else
 			return snprintf(dst, size, "%s/%s", xdg_ptr, name);
 	}
-#else
-	char *path_ptr = getenv("HOME");
-	if (path_ptr == NULL)
-		bcrash("Could not get $HOME\n");
-
-	if (!name || !*name)
-		return snprintf(dst, size, "%s", path_ptr);
-	else
-		return snprintf(dst, size, "%s/.%s", path_ptr, name);
-#endif
 }
 
 /* should return $HOME/.[name], or when using XDG,
  * should return $HOME/.config/[name] as default */
```

Comment:
If removing the XDG code path, should this comment be updated?

Reply:
Yes